bigquery/rest/README.md

```diff
@@ -0,0 +1,40 @@
+# Getting Started with BigQuery with the REST API
+
+Google's BigQuery Service features a REST-based API that allows developers to
+create applications to run ad-hoc queries on massive datasets. These sample
+Java applications demonstrate how to access the BigQuery API directly using the
+Google HTTP Client.
+
+## Quickstart
+
```

Comment:
Do you want to mention that they should update the constants if they do a verify?